src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -58,7 +58,7 @@ internal class SourcePropertyAccessorSymbol : SourceMemberMethodSymbol
                 syntax.Modifiers,
                 methodKind,
                 syntax.Keyword.IsKind(SyntaxKind.InitKeyword),
-                isAutoPropertyAccessor,
+                bodyShouldBeSynthesized: bodyShouldBeSynthesizedForSemicolonOnly && !hasBody && !hasExpressionBody,
```

Comment:
@AlekseyTs I'm not sure if this is the kind of thing you wanted.

Reply:
> I'm not sure if this is the kind of thing you wanted.

Yes, looks good. Thanks.